corehq/ex-submodules/pillowtop/migrations/0007_update_user_mapping_with_assigned_location_ids.py

```diff
@@ -0,0 +1,28 @@
+
+from django.db import migrations
+
+from corehq.pillows.mappings.user_mapping import USER_INDEX_INFO
+from corehq.util.django_migrations import update_es_mapping
+from corehq.apps.users.models import CouchUser
+from corehq.dbaccessors.couchapps.all_docs import get_all_docs_with_doc_types
+
+
+def all_webusers():
+    return get_all_docs_with_doc_types(db=CouchUser.get_db(), doc_types=["WebUser"])
+
+
+def reindex_web_users(*args, **kwargs):
+    for user in all_webusers():
+        CouchUser.wrap(user).save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pillowtop', '0006_add_geopoint_to_case_search_index'),
+    ]
+
+    operations = [
+        update_es_mapping(USER_INDEX_INFO.index),
+        migrations.RunPython(reindex_web_users),
```

Comment:
This operation does not have reversal logic.
```suggestion
        migrations.RunPython(
            reindex_web_users,
            reverse_code=migrations.RunPython.noop,
        ),
```